server/fishtest/templates/tests_view.mak

```diff
@@ -10,10 +10,13 @@
 
 % if show_task >= 0:
 <script>
-  document.documentElement.style="scroll-behavior:auto !important; overflow:hidden !important";
+  document.documentElement.style="scroll-behavior:auto; overflow:hidden;";
   function scroll_to(task_id) {
-    document.getElementById("task" + task_id)
-      .scrollIntoView();
+    task_offset = document.getElementById("task" + task_id).offsetTop;
```

Comment:
suggestions on little cleanup comments on the places that needs some cleanups as I didn't test the effects yet:
1- Variables should be declared as const or let.
2- We use camelCase for JavaScript and snake_case for python.